CHANGES.rst

```diff
@@ -10,6 +10,11 @@ charge_migration
 1.12.1 (2023-09-26)
 ===================
 
+associations
```

Comment:
Need to rebase this so that the new entry is up in the unreleased section of the log.